test_infra/ess/terraform.tf

```
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    ec = {
+      source  = "elastic/ec"
+      version = "0.10.0"
```

Why is this fixed to 0.10.0 when the latest release is 0.12.1 ? ---> https://registry.terraform.io/providers/elastic/ec/latest/docs maybe we can just specify a minimum version :
```suggestion
      version = ">= 0.10.0"
```